docs/DeveloperGuide.md

```diff
@@ -675,8 +693,39 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### **4. Enhanced Flexibility in Phone Number Parameter Input**
 
 ### **5. Find Feature Enhancement 1**
+**Potential Flaw in Current Implementation**<br>
+Currently, LinkMeIn only allows searching through the list of companies by the `COMPANY_NAME` parameter. However, 
+users might want to search through the list using other parameters, like `RECRUITER_NAME`, `PRIORITY` and `ROLE`.
+
+**Proposed Enhancement**<br>
+We plan to expand the current find command’s capability to allow for search using other parameters. The users will 
+be able to specify the prefix that corresponds to the parameter they wish to use for the search, before the keyword(s). 
+The prefixes used will be consistent with the rest of the application, in regard to what parameter they represent.
+
+Here are the new suggested formats :
+* Find using `RECRUITER_NAME` : `find n/KEYWORD`
+* Find using `PRIORITY`: `find pr/KEYWORD`
+* Find using `Role`: `find r/KEYWORD`
+
+**Examples**<br>
+* `find n/John Doe`
+* `find pr/High`
+* `find r/Software Engineer`
 
 ### **6. Find Feature Enhancement 2**
+**Potential Flaw in Current Implementation**<br>
+If users would like to find a specific company that has two or more words in their name such as `Microsoft 
+Corporation`, using the current find command will return companies that match either “Microsoft” or “Corporation”. 
+This can potentially pollute the results and defeat the purpose of the find feature.
+
+**Proposed Enhancement**<br>
+We plan to expand the find command's capability, to allow for exact keyword matching. This can be done by specifying 
+the keyword(s) within quotations.
+
+Suggested command format for exact find: `find “KEYWORD [KEYWORDS]...”`
+
+For example, users can now type: `find “Microsoft Corporation”`. And this will return results that match all the 
```

Comment:
For example, users can now type: `find “Microsoft Corporation”`. This will return companies with names that match "Microsoft Corporation" exactly, reducing the potential for polluted find results.